src/pip/__main__.py

```diff
@@ -3,6 +3,13 @@
 import os
 import sys
 
+# Remove '' and current working directory from the first entry
+# of sys.path, if present to avoid using current directory
+# in pip commands
```

Comment:
Mention "python -m pip" in this comment?

Reply:
Fixed!